### PR TITLE
intro docs

### PR DESCRIPTION
**Pull request by will-moore**
Started making some notes to help myself and others get started with understanding how the iviewer app fits together.
REAME viewable with diagram at https://github.com/will-moore/omero-iviewer/tree/intro_docs/docs
NB: need to update the URL to the diagram once this is good to merge, to use master instead of my branch.

E.g. viewer customisation request: https://www.openmicroscopy.org/community/viewtopic.php?f=4&t=8510

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+
+Application loading
+===================
+
+In views.py ``def index()`` loads variables from they query string,
```
Comment (jburel) [inline docs/README.md]: the

File: docs/README.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+	    let ctx = new Context(aurelia.container.get(EventAggregator), req);
+	    if (is_dev_server) ctx.is_dev_server = true;
+	    aurelia.container.registerInstance(Context,ctx);
+
+	    aurelia.start().then(
+	        () => aurelia.setRoot(PLATFORM.moduleName('app/index'), document.body));
+	});
+
+We create a singleton Context instance (app/context.js) which includes and EventAggregator (pub/sub events).
+This is registered with the aurilia's [dependency injection container](https://aurelia.io/docs/fundamentals/dependency-injection#explicit-configuration) to allow any component to get hold of it.
```
Comment (jburel) [inline docs/README.md]: aurilia -> aurelia

**Comment by jburel** [comment]
Thanks for starting it

**Comment by rgozim** [comment]
Looks okay. Can't say whether or not it's accurate or will help solve all iViewer problems, but I am sure we will find out when changes/fixes are made when the time comes.

**Comment by jburel** [comment]
Thanks @will-moore for the docs, very useful

**Pull request merged**